agent/metrics_test.go

```diff
@@ -94,30 +94,6 @@ func TestAgent_NewRPCMetrics(t *testing.T) {
 
 		assertMetricExists(t, respRec, metricsPrefix+"_rpc_server_call")
 	})
-
-	t.Run("Check that new rpc metrics can be filtered out", func(t *testing.T) {
```

Comment:
removing this since we are changing the default behavior for this new feature to be opt in by default (not opt out). I have a separate diff / patch that addresses the change but want a clean working tree in `main` in the meantime 💯